dsl_lang/lib/ast.ml

```diff
@@ -1,21 +1,17 @@
 type clss = string
 type var = string
 
-type declare =
-| DeclareClasses of clss list
-
 type policy =
 | Class of clss
 | Fifo of policy list
 | Fair of policy list
 | Strict of policy list
 | Var of var
 
-type return =
-| Return of policy
+type declare = clss list
+
+type assignment = var * policy
 
-type assignment =
-| Assn of var * policy
+type return = policy
 
-type program =
-| Prog of declare * (assignment list) * return
\ No newline at end of file
+type program = declare * (assignment list) * return
```

Comment:
Nice! Yeah we'll talk these back if we do need the ADTs, but these are lighter-weight and nicer for now

Reply:
Sounds good! As long as we keep the [slack thread](https://cucapra.slack.com/archives/C06CV424G94/p1721833098737389) on this topic open we should be good.